### ISSUE TITLE
Better ISSUE and PR templates

### ISSUE THREAD

**Issue by webimpress** [zendframework/maintainers#56]
We can now have better ISSUE and PR templates on GitHub, please see for example it here:

https://github.com/doctrine/doctrine2/issues/new/choose

I think it would be nice to update it also in ZF org :) Thoughts? Suggestions?

**Comment by froschdesign** [comment]
Questions:

* Who can create these templates? (the template creator is only available in the settings)
* Is there an API endpoint for this?

**Comment by webimpress** [comment]
@froschdesign Hm... It looks like these are just normal files in the repository:

https://github.com/doctrine/doctrine2/tree/master/.github

So I guess, we would need first templates created here, approved here - in that repo and then we can spread it to all repos :)

**Comment by froschdesign** [comment]
@webimpress 
I read this in the help section but did not find the folder. 🤦‍♂️

**Comment by froschdesign** [comment]
> in that repo and then we can spread it to all repos.

via API!

**Comment by webimpress** [comment]
> via API!

Of course, I can't imagine do it manually for all repos :) I've done it already couple times so it's not a problem at all. We just need approved templates
